enhancements/two-nodes-openshift/2no.md

```diff
@@ -119,12 +119,24 @@ When starting etcd, the OCF script will use etcd's cluster ID and version counte
 
 #### Cluster Creation
 
-User creation of a two-node control-plane will be possible via the Assisted Installer.
+User creation of a two-node control-plane will be possible via the Assisted Installer. A key requirement of the project is that it can be deployed using only 2 nodes, so deployments will take advantage of the Assisted Installer's ability to use one of the target machines as the bootstrap node before it is rebooted into a control-plane node. There is a critical transition during this process, where in order to maintain etcd quorum, the bootstrap node will need to be removed from the etcd cluster before it is rebooted so that quorum can be maintained as the machine reboots into a second control-plane.
 
-The procedure follows the standard flow except for the configuration of 2 nodes instead of 3, and the collection of RedFish details (including passwords!) for each node which are needed for the RHEL-HA configuration.
-If available via the SaaS offering (not confirmed, ZTP may be the target), the offering will need to ensure passwords are appropriately handled. 
+Otherwise, the procedure follows the standard flow except for the configuration of 2 nodes instead of 3. At this time we've discussed the collection of RedFish details (including passwords!) for each node. This is needed for the RHEL-HA configuration by leveraging the BareMetalHost CRDs populated from the baremetal platform specification in the install-config. There are open questions on how to ensure that pacemaker is the only entity responsible for fencing to prevent conflicting requests from the baremetal operator.
```

Comment:
Regarding the questions about pacemaker being the only fencing entity, do you refer to RHWA operators  ? (i.e NHC, SNR, FAR etc...)

Reply:
I was making reference to BMO - which doesn't manage fencing but it does care about power management for nodes.

Reply:
I added a line to note the optional operators.